ui/src/app/base/components/FormikForm/FormikForm.tsx

```diff
@@ -23,7 +23,7 @@ export type Props<V, E = FormErrors> = {
   };
   onSubmit: (values?: V, formikHelpers?: FormikHelpers<V>) => void;
   savedRedirect?: string;
-  validationSchema?: SchemaOf<V>;
+  validationSchema?: AnySchema;
```

Comment:
This was causing a typescript error for any `FormikForm` that was given a generic

Reply:
I think the previous type was correct, but looks like you have to do a little bit more setup to define the type of the Schema:

```
  const PowerFormSchema: SchemaOf<PowerFormValues> = Yup.object()
    .shape({
      powerParameters: Yup.object().shape(powerParametersSchema),
      powerType: Yup.string().required("Power type is required"),
    })
    .defined();
```
From: https://github.com/jquense/yup/blob/master/docs/typescript.md

Reply:
Aaaaaaahhhhh I missed the `defined()` bit when I was looking at that earlier. Nice, thanks, I'll revert the change